build.gradle

```diff
@@ -284,8 +284,8 @@ subprojects { subproject ->
         testCompile group: 'org.powermock', name: 'powermock-api-mockito2', version: powermockVersion
         testCompile group: 'org.powermock', name: 'powermock-module-junit4', version: powermockVersion
 
-        testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.19.3'
-        testCompile group: 'com.github.hmcts', name: 'befta-fw', version: '8.7.2-Categories'
+        testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.19.5_HMAN-368'
+        testCompile group: 'com.github.hmcts', name: 'befta-fw', version: '8.7.6-HMAN-368'
```

Comment:
These should be updated to match final version numbers if being merged to `master`.